resources/web/styles.css

```diff
@@ -28,51 +28,20 @@
   position: relative;
 }
 
-/* Format floating role=xpack titles */
-#guide h1 > span[class~="xpack"],
-#guide h2 > span[class~="xpack"],
-#guide h3 > span[class~="xpack"],
-#guide h4 > span[class~="xpack"],
-#guide h5 > span[class~="xpack"],
-#guide h6 > span[class~="xpack"] {
-  display: inherit;
-  font-size: inherit;
-  margin-top: inherit;
-  margin-bottom: inherit;
-  margin-left: inherit;
-  margin-right: inherit;
-  font-weight: inherit;
-}
-
-/* Add xpack icon to role=xpack sidebar titles */
-#guide div[class~="xpack"] > div > div > div > p > strong:after,
-
-/* Add xpack icon to role=xpack dt titles */
-#guide dt > span > span.xpack:after,
```

Comment:
Do we have any definitions that are just x-pack? If so I'll need to do something for them.

Reply:
I didn't find any in the stack-docs, kibana, or elasticsearch repos